include/sway/output.h

```diff
@@ -73,6 +73,7 @@ struct sway_output {
 	int max_render_time; // In milliseconds
 	struct wl_event_source *repaint_timer;
 	bool gamma_lut_changed;
+	bool allow_tearing, can_tear;
```

Comment:
Maybe rename `can_tear` to `enable_tearing`?

Reply:
Renamed.